src/routes/changeRequest.ts

```diff
@@ -89,6 +90,45 @@ export const createChangeRequest = async (req, res, person, method, delta) => {
   });
 };
 
+export const createBusinessChangeRequest = async (
```

Comment:
let's skip ChangeRequest logic for now. We do not know yet how to handle it for business, because devices are assigned to physical persons and not to business